config-templates/redhat/odf-local/changelog.json

```diff
@@ -3,43 +3,43 @@
         "4.13": {
             "status": "supported",
             "latest-revision": {
+                "revision": "6",
+                "release-date": "19 January 2023",
+                "cves": "",
+                "golang": "1.20.11",
+                "ubi": "8.9-1029",
```

Comment:
new ubi image is available. Do you want to upgrade to new UBI image and rebuild the image?

Reply:
will do that in the next push along with VPC and classic update